ui/frontend/build_src/src/components/organisms/creationPanel/basicCreation/makeButton/index.tsx

```diff
@@ -1,44 +1,175 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
```

Comment:
this is the file that got most of the additions. This is basically all the code from the main js file.